iceoryx_examples/iceoptions/README.md

```diff
@@ -81,6 +84,18 @@ If we want to enforce the contract that the publisher needs to support a `histor
 `requirePublisherHistorySupport` to `true`. In this case the subscriber will only connect if the publisher history support is at least as large as is request.
 By default this is set to `false` and best-effort behavior is used.
 
+!!! warning
+    In case of n:m communication, the history feature will **not** provide the overall last n samples based on delivery point in time!
+
+    The following two scenarios are examples of issues when using n:m together with the history feature:
+
+    1. Multiple publishers
+    The last n samples of every publisher are received. This means for m publishers in the worst
+    case m * n samples in random order, not in the order they were published on the topic.
+
+    2. Multiple publishers after the publisher called `stopOffer()` or is removed
+    The last n samples will never be received since they vanished. An arbitrary number of samples or nothing is received.
```

Comment:
For me this important information does not belong into the example but in the `qos-policies` under a `Limitations` section. This could be linked from here but the information should be there.

Reply:
I think it's more on the spot in the example as more people will look at the example than into the QoS article. I added the warning to the QoS and linked to `iceoptions`. Is that ok for you?